cli/templatepush.go

```diff
@@ -110,6 +117,20 @@ func (pf *templateUploadFlags) checkForLockfile(inv *clibase.Invocation) error {
 	return nil
 }
 
+func (pf *templateUploadFlags) templateMessage(inv *clibase.Invocation) string {
+	title := strings.SplitN(pf.message, "\n", 2)[0]
+	if len(title) > 72 {
+		cliui.Warn(inv.Stdout, "Template message is longer than 72 characters, it will be displayed as truncated.")
+	}
+	if title != pf.message {
+		cliui.Warn(inv.Stdout, "Template message contains newlines, only the first line will be displayed.")
+	}
+	if pf.message != "" {
+		return pf.message
+	}
+	return "Uploaded from the CLI"
```

Comment:
I'm not sure about using this default message when it is empty/missing. Thoughts?

Reply:
I would for an analogy to Git commits. Usually, it is expected to provide something, so maybe we should stick to this logic here. What I mean:
* forbid empty messages
* use the default `Uploaded from the CLI`/`Uploaded from the UI` if empty.

Reply:
That's essentially the logic now since it's not possible to set it to empty. I'm guessing who modified it should already be part of the template version and stored in the DB so I don't think it needs to be added here, good thinking though.

However, I started to dislike "Uploaded from the CLI" by default since it's not really any better than an empty message and doesn't convey the purpose of having it.

Perhaps the Git way of forcing a manually written message or using `--allow-empty` would make sense, but that's a breaking change. Thoughts @bpmct?